clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/ServerGroup.groovy

```diff
@@ -122,6 +123,7 @@ interface ServerGroup {
   @JsonIgnore
   ImageSummary getImageSummary()
 
+  @JsonPropertyOrder(alphabetic = true)
```

Comment:
It would be nice if we were able to configure the `ObjectMapper` used to serialize results - I think you can use `MapperFeature.SORT_PROPERTIES_ALPHABETICALLY` in that case to do the same thing. But I'm not sure where that ObjectMapper lives...